tests/frontend/github_repo_info_controller_test.js

```diff
@@ -41,13 +41,19 @@ const mountDom = async () => {
           <span data-github-repo-info-target="openIssuesCount"></span>
         </a>
       </li>
+      <li>
+        <a data-github-repo-info-target="openPRsUrl">
+          <span data-github-repo-info-target="openPRsCount"></span>
+        </a>
+      </li>
     </div>
   `;
   document.body.innerHTML = `
     <div id="github-repo-stats"
           data-controller="github-repo-stats"
           data-github-repo-stats-github-repo-info-outlet=".github-repo-info">
           data-github-repo-stats-url-value="https://api.github.com/repos/pypi/warehouse">
+          data-github-repo-stats-issueUrl-value="https://api.github.com/search/issues?q=repo:pypi/warehouse+type:issue+state:open&per_page=1">
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏This URL prefix does not currently exist in the [Content Security Policy](https://github.com/pypi/warehouse/blob/8218d5d9e6f72d90bd34cc64ca2cf25aff87a471/warehouse/csp.py#L88) so it will be blocked by browsers. Please add it to the scope (and update associated tests).